meeting-format.md

```diff
@@ -0,0 +1,15 @@
+# Format of Meetings
+
+1. Each meeting should in general be 1 hour.  Chairpeople should take a vote as to whether the meeting should be extended at the 50 minute mark.
+
+2. Meetings should begin with introductions by each particpant.
+
+3. Meetings should generally be focused around discussing solutions to one or two specific problems.
+
+4. Discussion should be focused around enumerating possible solutions, and the positives and negatives of each solution.
+
+5. Meetings should end with the chairperson determining:
+	
+    a. If there are any action items, and who is taking responsibility for each.
```

Comment:
The chairperson need not necessarily be the note taker, but maybe include a recommendation for the chairperson to either take notes themselves or appoint a note taker.  Same goes for recordings, as they're a form of notes.  These help improve determination of action items.

Reply:
This looks like text you want me to include somewhere in the document, but it's not clear to me where.    

Something similar is covered under: https://github.com/bitcoincashorg/workgroups/blob/master/workgroup-positions.md

Maybe you could submit another pull-request? (More contributors the merrier! :D)

Reply:
Ya, sorry. I didn't have an exact place in mind.  That document appears to cover it well enough though. 👍